src/main/java/com/smartcar/sdk/ApiClient.java

```diff
@@ -34,7 +34,20 @@
     .readTimeout(300, TimeUnit.SECONDS)
     .callTimeout(360, TimeUnit.SECONDS)
     .build();
-  static GsonBuilder gson = new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES);
+
+  private static String toCamelCase(String fieldName) {
```

Comment:
maybe we can use a built in function, https://commons.apache.org/proper/commons-text/apidocs/org/apache/commons/text/CaseUtils.html#toCamelCase(java.lang.String,boolean,char...)